engine/server/sv_save.c

```diff
@@ -2433,10 +2434,10 @@ int GAME_EXPORT SV_GetSaveComment( const char *savename, char *comment )
 
 		// split comment to sections
 		if( Q_strstr( savename, "quick" ))
-			Q_strncat( comment, "[quick]", CS_SIZE );
+			len = Q_strncpy( comment, "[quick]", CS_SIZE );
```

Comment:
Again, is there any evidence that string length tracking is better than just calling strncat a few times in a function that's not performance critical or has string concatenation bottleneck?

Also, once you're here, `sizeof( comment )`